src/ric/d_19C70.c

```diff
@@ -0,0 +1,114 @@
+#include "ric.h"
+
+s16 D_80155C70[] = {0x8139, 0x813A, 0x813B, 0x0000};
+
+s16 D_80155C78[] = {0x0010, 0x000E, 0x000C, 0x000A, 0x0008, 0x0006,
+                    0x0004, 0x0002, 0x000F, 0x000D, 0x000B, 0x0009,
+                    0x0007, 0x0005, 0x0003, 0x00FF};
+
+u16 D_80155C98[] = {0x0000, 0x0002, 0x0004, 0x0006, 0x0008, 0x000A,
+                    0x000C, 0x000E, 0x0001, 0x0003, 0x0005, 0x0007,
+                    0x0009, 0x000B, 0x000D, 0x000F};
+
+s16 D_80155CB8[] = {0x000A, 0x000B, 0x000D, 0x0011, 0x000F,
+                    0x000C, 0x0012, 0x000E, 0x0010, 0x0000};
+
+s16 D_80155CCC[] = {0x000A, 0x000B, 0x0011, 0x000D, 0x000F,
+                    0x0012, 0x000C, 0x0010, 0x000E, 0x0000};
+
+s16 D_80155CE0[] = {0x0001, 0x0002, 0x0004, 0x0008, 0x0006,
+                    0x0003, 0x0009, 0x0005, 0x0007, 0x0000};
+
+s16 D_80155CF4[] = {0x0001, 0x0002, 0x0008, 0x0004, 0x0006,
+                    0x0009, 0x0003, 0x0007, 0x0005, 0x0000};
+
+s16 D_80155D08[] = {0x0013, 0x0014, 0x0016, 0x001A, 0x0018,
+                    0x0015, 0x001B, 0x0017, 0x0019, 0x0000};
+
+s16 D_80155D1C[] = {0x0013, 0x0014, 0x001A, 0x0016, 0x0018,
+                    0x001B, 0x0015, 0x0019, 0x0017, 0x0000};
+
+s16 D_80155D30[] = {
+    0x0010, 0x0018, 0x0011, 0x0019, 0x0012, 0x001A, 0x0013, 0x001B, 0x0014,
+    0x001C, 0x0015, 0x001D, 0x0016, 0x001E, 0x0017, 0x0000, 0x0002, 0x0001,
+    0x0002, 0x0002, 0x0002, 0x0003, 0x0002, 0x0004, 0x0000, 0x0000};
+
+s16 D_80155D64[4][6] = {
+    {
+        0x0002,
+        0xFFFE,
+        0x0000,
+        0xFFFC,
+        0x0000,
+        0x0000,
```

Comment:
```suggestion
        2, -2, 0, -4, 0, 0,
```
all the values from this array are signed